src/components/books/Book.module.css

```diff
@@ -1,3 +1,35 @@
 .book {
-  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+  background-color: white;
+  width: 85%;
+  padding: 20px;
+  padding-inline: 40px;
+}
+
+.title_author {
+  width: 40%;
+  border-right: 1px solid rgb(68, 67, 67);
+}
+
+.title_author h3 {
+  font-size: 25px;
+}
+
+.title_author p {
+  font-size: 17px;
+  color: #037cd8;
+}
+
+button {
+  border: none;
+  padding-inline: 20px;
+  padding-block: 10px;
+  background-color: #0290ff;
+  color: white;
+  font-size: 15px;
+  border-radius: 3px;
+  cursor: pointer;
 }
```

Comment:
### Your Project Design 🔴
![Capture - Copy](https://user-images.githubusercontent.com/93039359/208041947-00ddf727-a2e7-4883-bdb6-aa43cb4c8121.PNG)

### Required Project Design 🟢
![bookstore](https://user-images.githubusercontent.com/93039359/208042676-b83eb703-7e55-42cb-93f0-f7598c3cc8b1.png)

**[Mandatory Change]** 📝
- [ ] You can see that your `Add` button is not of a proper size according to the design 😨 Kindly follow the design to make your project professional 👍

**[Optional Change]** 📂
- [ ] It would be nice if your form would reset after adding a book to make your application behaviour familiar for the users 😊